packages/opentelemetry-cloud-trace-exporter/src/types.ts

```diff
@@ -12,26 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Compute, JWT, OAuth2Client } from 'google-auth-library';
+import { Metadata } from 'grpc';
 
 export interface Span {
   name?: string;
   spanId?: string;
   parentSpanId?: string;
   displayName?: TruncatableString;
-  startTime?: string;
-  endTime?: string;
+  startTime?: Timestamp;
+  endTime?: Timestamp;
   attributes?: Attributes;
   // This property is currently unused. keeping it here as it is part
   // of the stack driver trace types and may be used in the future
   stackTrace?: StackTrace;
   timeEvents?: TimeEvents;
   links?: Links;
   status?: Status;
-  sameProcessAsParentSpan?: boolean;
+  sameProcessAsParentSpan?: BoolValue;
   childSpanCount?: number;
 }
 
+export interface Timestamp {
+  seconds?: number;
+  nanos?: number;
```

Comment:
Is there a place where these are undefined?

Reply:
I just made it consistent with the original definition of a timestamp where all fields are optional. Since we always set their value, I can make them mandatory here